### ISSUE TITLE
Change 'all contributions' checkbox in the webcast/recording request form

### ISSUE THREAD

**Issue by javfg** [indico/indico-plugins-cern#90]
When selecting the contributions for a webcast/recording request, there is an 'All contributions' checkbox, but it is not flexible enough:

* User might want to check all but a few contributions in a huge list.
* User might want to record virtual contributions (or held in a room with no video-conference equipment)

One solution could be changing it into a button that checks/unchecks all the contributions.

**Comment by ThiefMaster** [comment]
> One solution could be changing it into a button that checks/unchecks all the contributions.

Not really, checking "all contributions" and actively checking everything has different semantics:

- "all contributions" means all *compatible* contributions (ie in a proper room), including those added in the future
- checking specific contributions may override the compatibility check (if done by service managers / admins), and also only selects those particular contributions

**Comment by ThiefMaster** [comment]
INHO the ideal solution would be to make the compatibility check softer and either require a compatible room OR the user confirming that it's a virtual event. However, I think there was some reason why we didn't want to make it even easier to request recording for virtual events...

@tgbaron do you remember? If it was about putting too much load on the recording service, maybe we could have such a checkbox anyway but add a message telling people that depending on the event no video cutting services etc. are provided?
